custom.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "*.png" {
+    const value: any;
+    export default value;
+}
```

Comment:
no extra line at the end of file D:
Also this might look better as index.d.ts,
Can you explain what this type fixes or does?

Reply:
Checked out nextjs docs and found the image component that I just imported instead which removes the need for this custom type.

Reply:
Even better :)